### DISCUSSION TITLE
DraftHead

### DISCUSSION THREAD

**Discussion by PrinceZaman-2005** [PrinceZaman-2005/DraftHead#4] [Announcements]
**This is the upgraded version of DraftHead!**
DraftHead is a experimental project inspired by modern drafting architectures, investigating importance-weighted bias additions to transformer weights by generating several possible thoughts, estimate confidence for each thought, and let attention naturally reinforce high-confidence with backprop.

#Improvements:
1. Added noise injection to prevent overfitting
2. selective drafts only (top 2)
3. Used more dataset to generalize.
4. Test accuracy improved

<img width="1013" height="527" alt="Screenshot 2026-09-19 210210" src="https://github.com/user-attachments/assets/e40fef4b-fd8e-4d39-8b8a-ec9f5241837e" />

<hr /><em>This discussion was created from the release <a href='https://github.com/PrinceZaman-2005/DraftHead/releases/tag/'>DraftHead</a>.</em>
